src/Http/Role/DeleteConstraintRequestHandler.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Http\Role;
+
+use Broadway\CommandHandling\CommandBus;
+use CultuurNet\UDB3\Http\Request\RouteParameters;
+use CultuurNet\UDB3\Model\ValueObject\Identity\UUID;
+use CultuurNet\UDB3\Role\Commands\RemoveConstraint;
+use Fig\Http\Message\StatusCodeInterface;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Server\RequestHandlerInterface;
+use Slim\Psr7\Response;
+
+class DeleteConstraintRequestHandler implements RequestHandlerInterface
+{
+    private CommandBus $commandBus;
+
+    public function __construct(CommandBus $commandBus)
+    {
+        $this->commandBus = $commandBus;
+    }
+
+    public function handle(ServerRequestInterface $request): ResponseInterface
+    {
+        $routeParameters = new RouteParameters($request);
+        $roleId = new UUID($routeParameters->get('id'));
```

Comment:
In your previous pull request for the reading part, you introduced `getRoleId`. Maybe reuse that?

Reply:
That will require me to change the routes. I was planning on doing that when I made the switch to requestHandlers in the routing provider.

Reply:
Come to think of it, I can already implement it here, as it's not being used anyway. How do we feel about that getRoleId returning a UUID VO and throwing an API problem when it's invalid. In other places it already throws an invalidArgumentException because of this. Seems like it conceptually belongs on the routeParameters class.

cc @bertramakers

Reply:
I think this code path is safe to use `UUID` as the typed return type because roles are "new" in UDB3 and will always match the `UUID` pattern. On other paths for example labels and events can have route parameters that don't match their VO.

Reply:
All right, I'll make that change in the last PR